GNUmakefile

```diff
@@ -397,6 +397,17 @@ cspc-operator-image:
 	@cd buildscripts/${CSPC_OPERATOR} && sudo docker build -t ${HUB_USER}/${CSPC_OPERATOR_REPO_NAME}:${IMAGE_TAG} --build-arg BUILD_DATE=${BUILD_DATE} .
 	@rm buildscripts/${CSPC_OPERATOR}/${CSPC_OPERATOR}
 
+.PHONY: cspc-operator-debug-image
+cspc-operator-debug-image:
+	@echo "----------------------------"
+	@echo -n "--> cspc-operator image "
+	@echo "${HUB_USER}/${CSPC_OPERATOR_REPO_NAME}:${IMAGE_TAG}"
+	@echo "----------------------------"
+	@PNAME=${CSPC_OPERATOR} CTLNAME=${CSPC_OPERATOR} BUILD_TAG="-tags=debug" sh -c "'$(PWD)/buildscripts/build.sh'"
+	@cp bin/${CSPC_OPERATOR}/${CSPC_OPERATOR} buildscripts/cspc-operator-debug/
+	@cd buildscripts/cspc-operator-debug && sudo docker build -t ${HUB_USER}/${CSPC_OPERATOR_REPO_NAME}:${IMAGE_TAG} --build-arg BUILD_DATE=${BUILD_DATE} .
```

Comment:
have to add the image to `deploy-images` target to push the image ?

Reply:
What will be the tag `ci` ?

Reply:
IMO let's have the `CTLNAME` directory here to have a different path for bebug builds as the binary name is same ?